plotting/operationalPlots.py

```diff
@@ -343,32 +345,9 @@ def plotSpatialFieldData(contourpacket, fieldpacket, prefix='', nested=True, mod
         if 'directions' in kwargs:
             Us = -np.rad2deg(np.cos(np.deg2rad(kwargs['directions'][tt])))
             Vs = -np.rad2deg(np.sin(np.deg2rad(kwargs['directions'][tt])))
-            xp = xcoord
```

Comment:
I think all of this needs to stay, unless you changed it.  Chuan had done a lot of work here over the summer

Reply:
**separate merge**